cmd/cli/kubectl-kyverno/policy/load.go

```diff
@@ -125,6 +121,20 @@ func kubectlValidateLoader(path string, content []byte) (*LoaderResults, error)
 		return nil, err
 	}
 	results := &LoaderResults{}
+
+	crds, err := data.Crds()
+	if err != nil {
+		return nil, err
+	}
+
+	factory, err := resourceloader.New(openapiclient.NewComposite(
+		openapiclient.NewHardcodedBuiltins("1.28"),
```

Comment:
i wonder if we want to change the `1.28` here 🤔

Reply:
Good catch! Will push the change.